alcs-frontend/src/app/features/home/gis/gis-subtasks.component.ts

```diff
@@ -39,14 +39,17 @@ export class GisSubtasksComponent implements OnInit {
     const applications = nonOrderedSubtasks.filter((s) => s.card.type === CardType.APP);
     const reconsiderations = nonOrderedSubtasks.filter((s) => s.card.type === CardType.RECON);
     const planningReviews = nonOrderedSubtasks.filter((s) => s.card.type === CardType.PLAN);
+    const amendments = nonOrderedSubtasks.filter((s) => s.card.type === CardType.AMEND);
 
     this.subtasks.push(
       // high priority
       ...applications.filter((a) => a.card.highPriority).sort((a, b) => b.activeDays! - a.activeDays!),
+      ...amendments.filter((r) => r.card.highPriority).sort((a, b) => a.createdAt! - b.createdAt!),
```

Comment:
so amendments need to show up before recons?

Reply:
correct, as per the ticket amendments should be below applications and above recons.